packages/sdk/client/src/packlets/client/iframe-service-proxy.ts

```diff
@@ -196,10 +196,13 @@ export class IFrameClientServicesProxy implements ClientServicesProvider {
     );
 
     if (!this._iframe) {
-      const res = await fetch(source);
-      if (res.status >= 400) {
-        throw new RemoteServiceConnectionError(`Failed to fetch ${source}`, { source, status: res.status });
-      }
+      // TODO(wittjosiah): This doesn't work with generated service worker when offline. Remove?
+      //   Is there an easy way to respond to this without a lot of custom work inside the service worker?
+      //   Is getting an http response here helpful information or just overhead?
```

Comment:
+1 for removing, if there's an http error when loading the iframe, you'll get a console message anyway

Reply:
I'm not sure you'd get a console message because it's not a fetch but just iframe being loaded. It's definitely visible in the network tools though.